src/main/java/Pac.java

```diff
@@ -0,0 +1,229 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Pac {
+    static String logo = " ____     ___    _____\n"
+            + "|  _ \\   / _ \\  |  ___|\n"
+            + "| |_| | | |_| | | |\n"
+            + "|  __/  | | | | | |___\n"
+            + "|_|     |_| |_| |_____|\n";
+    static String newline = "----------------------------------------------------";
```

Comment:
Should the logo and newline variables be made final? And their names changed to fit the condone standard for naming constants eg: LOGO